app/models/corpus_item_model.py

```diff
@@ -3,4 +3,5 @@
 
 class CorpusItemModel(BaseModel):
     id: str
+    publisher: str
```

Comment:
Should this be optional? I'm not sure if we can guarantee to have a publisher for each item? We definitely don't have it backfilled.

Reply:
and now it is....